shared_model/interfaces/iroha_internal/transaction_batch_helpers.hpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_TRANSACTION_BATCH_HELPERS_HPP
+#define IROHA_TRANSACTION_BATCH_HELPERS_HPP
+
+#include <sstream>
+
+#include "cryptography/hash.hpp"
+
+namespace shared_model {
+  namespace interface {
+
+    /**
+     * Provides a method that calculates reduced batch hash
+     */
+    class TransactionBatchHelpers {
```

Comment:
Why make a class with only static functions?

Reply:
This is for compatibility reasons, since it was previously in TransactionBatch class. It can probably be made a free function in a separate task.